catalog/MDCCatalog/MDCCatalogTiles.m

```diff
@@ -4098,12 +4109,19 @@ void MDCCatalogDrawTabsTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme)
   CGColorSpaceRelease(colorSpace);
 }
 
-static NSString* const TextFieldFontName = @"RobotoMono-Medium";
-
 void MDCCatalogDrawTextFieldTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme) {
```

Comment:
Roboto Mono should not be removed from the text fields tile. It's a design choice.